### PR TITLE
Update UITour documentation for Firefox 40

### PR DESCRIPTION
**Pull request by alexgibson**
I still need to verify that this is indeed all correct before merging.

### CODE REVIEW

**Comment by alexgibson** [comment]
Ok I believe the info here is all correct / accurate. Should be good for review.

File: docs/uitour.rst [hunk 1]
```diff
@@ -308,7 +314,9 @@ The ``defaultUpdateChannel`` key has many possible values, the most important be
 
 .. Important::
 
-    ``appinfo`` is only available in Firefox 35 onward.
+    ``appinfo`` is only available in Firefox 35 onward. ``defaultBrowser`` will only be returned on Firefox 40 or later.
+
```
Comment (schalkneethling) [inline docs/uitour.rst]: Should this be `and`?

File: docs/uitour.rst [hunk 2]
```diff
@@ -529,6 +554,7 @@ Event types:
 * ``'Loop:IncomingConversation'`` - User has an incoming conversation. Event will have data boolean value ``conversationOpen`` set to ``true`` or ``false`` depending on if the chat window is open or not.
 * ``'Loop:RoomURLCopied'`` - User clicks the copy button to share a chat URL.
 * ``'Loop:RoomURLEmailed'`` - User clicks the email button to share a chat URL.
+* ``'Loop:RoomURLShared'`` - User clicks the share button to share a chat URL.
 * ``'Loop:PanelTabChanged'`` - User clicks on the Contacts or Room tab in the panel. Event will return data = ``rooms`` or ``contacts`` depending on which tab the user clicked on.
```
Comment (schalkneethling) [inline docs/uitour.rst]: Nit: should this be ``data = rooms`` or ``contacts``?

**Comment by alexgibson** [comment]
@schalkneethling - updated thanks

**Comment by schalkneethling** [comment]
r+ :beer:

**Pull request merged**